lib/utils.pm

```diff
@@ -135,6 +135,13 @@ sub wait_boot {
     if ($textmode || check_var('DESKTOP', 'textmode')) {
         assert_screen 'linux-login', 200;
         reset_consoles;
+
+        # Without this login name and password won't get to the system. The
+        # get lost somewhere. Applies for all systems installed via svirt.
+        if (check_var('BACKEND', 'svirt') and !check_var('ARCH', 's390x')) {
+            wait_idle;
```

Comment:
You know we want to get rid of `wait_idle` as it doesn't scale well and is not available on most architectures. Better wait for a real event e.g. use `assert_screen` or `wait_serial` or - if you must - some backend specific wait call e.g. as for s390x zkvm: `expect_s3270`